File: CPP/Barang.cpp
#include <string>
using namespace std;

class Barang {
private:
    int id;
    string nama;
    string kategori;
    double harga;
    int stok;

public:
    // Constructor
    Barang(int id, string nama, string kategori, double harga, int stok) {
        this->id = id;
        this->nama = nama;
        this->kategori = kategori;
        this->harga = harga;
        this->stok = stok;
    }

    // Getter
    int getId() { return id; }
    string getNama() { return nama; }
    string getKategori() { return kategori; }
    double getHarga() { return harga; }
    int getStok() { return stok; }

    // Setter
    void setNama(string nama) { this->nama = nama; }
    void setKategori(string kategori) { this->kategori = kategori; }
    void setHarga(double harga) { this->harga = harga; }
    void setStok(int stok) { this->stok = stok; }
};

File: JAVA/Barang.java
public class Barang {
    private int id;
    private String nama;
    private String kategori;
    private double harga;
    private int stok;

    public Barang(int id, String nama, String kategori, double harga, int stok) {
        this.id = id;
        this.nama = nama;
        this.kategori = kategori;
        this.harga = harga;
        this.stok = stok;
    }

    public int getId() { return id; }
    public String getNama() { return nama; }
    public String getKategori() { return kategori; }
    public double getHarga() { return harga; }
    public int getStok() { return stok; }

    public void setNama(String nama) { this.nama = nama; }
    public void setKategori(String kategori) { this.kategori = kategori; }
    public void setHarga(double harga) { this.harga = harga; }
    public void setStok(int stok) { this.stok = stok; }
}